wiki/grips-and-tricks.md

```diff
@@ -19,16 +19,37 @@ Additionally, here's an image from the game developers about how the
 
 ![Bloq Hitbox](~@images/mapping/hitbox-from-split.jpg)
 
-## Ranking
+## Ranking Systems
```

Comment:
Would it be better to keep as Ranking to keep link compatibility?

Reply:
would updating everything that links to this header fix compatibility or just revert?

Reply:
revert since can't practically update links people made outside of the wiki. 

Would want to preserve unless it doesn't make sense in the context of the section which I think leaving as is still works